src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -67,7 +71,8 @@ public CheckInProcessRecords withLoan(Loan loan) {
       loan,
       this.requestQueue,
       this.checkInServicePoint,
-      this.highestPriorityFulfillableRequest);
+      this.highestPriorityFulfillableRequest,
+        this.loggedInUserId);
```

Comment:
GitHub shows the indentation is different, why is that the case?

Reply:
Done.